service/test/integration2_test.go

```diff
@@ -865,7 +865,7 @@ func TestClientServer_traceContinue(t *testing.T) {
 					t.Fatalf("No goroutine information")
 				}
 
-				if len(bpi.Stacktrace) == 0 {
+				if len(bpi.Stacktrace) <= 0 {
```

Comment:
This is undoing a recent commit.

Reply:
Thanks for the catch have corrected it now